Dockerfile

```diff
@@ -37,7 +37,7 @@ ENV DEVISE_SECRET boopboop
 ENV SECRET_COOKIE_TOKEN beepbeep
 
 # build assets
-RUN bundle exec rake deploy:build
+RUN bundle exec rake deploy:build docker
```

Comment:
I did not try, that was a quick draft after browsing the code, to check if I was going in the right direction :-)